### PR TITLE
eslint: add eslint:recommended and fix a few more issues

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

File: src/bidiMapper/domains/script/scriptEvaluator.ts [hunk 1]
```diff
@@ -306,11 +306,11 @@ function webDriverValueToBiDi(
   }
 
   if (result.type == 'node') {
-    if (bidiValue.hasOwnProperty('backendNodeId')) {
+    if (Object.prototype.hasOwnProperty.call(bidiValue, 'backendNodeId')) {
```
Comment (OrKoN) [inline src/bidiMapper/domains/script/scriptEvaluator.ts:309]: why not https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn ?

Comment (thiagowfx) [reply src/bidiMapper/domains/script/scriptEvaluator.ts:309]: Sure! My fix was the recommended one in the rule README page, but that should also work.

Comment (thiagowfx) [reply src/bidiMapper/domains/script/scriptEvaluator.ts]: Done.

Except here:

src/bidiMapper/domains/log/logHelper.ts:154:12 - error TS2339: Property 'hasOwn' does not exist on type 'RemoteValue'.
  Property 'hasOwn' does not exist on type 'UndefinedValue'.

File: src/bidiMapper/domains/log/logHelper.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ function toJson(arg: CommonDataTypes.RemoteValue): string {
 }
 
 function stringFromArg(arg: CommonDataTypes.RemoteValue): string {
-  if (!arg.hasOwnProperty('value')) {
+  if (!Object.prototype.hasOwnProperty.call(arg, 'value')) {
```
Comment (OrKoN) [inline src/bidiMapper/domains/log/logHelper.ts:154]: same here

Comment (thiagowfx) [reply src/bidiMapper/domains/log/logHelper.ts]: Done

**Review by OrKoN — APPROVED** [review]

**Pull request merged**